Components/SearchBar.js

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { Searchbar } from 'react-native-paper';
+
+const searchBar = () => {
+  const [searchQuery, setSearchQuery] = React.useState('');
+
+  const onChangeSearch = query => setSearchQuery(query);
```

Comment:
onChangeSearch is functionally identical to setSearchQuery, this assignment seems unnecessary 🤔 Just use setSearchQuery on line 12

Reply:
Re: my larger review, this function should actually be eliminated within your revisions, but still a good thing to pay attention to this in the future.